java_socketio_chatroom_client/src/main/java/io/github/marcodiri/java_socketio_chatroom_client/App.java

```diff
@@ -15,10 +15,14 @@
 
 	public static void main(String[] args) {
 		LOGGER.info("Client app started");
+		String serverHost = "localhost";
+		if (args.length > 0)
+			serverHost = args[0];
+		String finalServerHost = serverHost;
```

Comment:
Why another variable? Isn't just serverHost enough?

Reply:
Even for me it's strange but this is the compilation error on line 25: `Variable used in lambda expression should be final or effectively final`.
In practice, must be passed to it a variable that doesn't change value